### ISSUE TITLE
Null-coalescing operator usage warning on clean code - [int] vs [float]

### ISSUE THREAD

**Issue by badmansan** [kalessil/phpinspectionsea#1643]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) |
| Language level | PHP 7.2                        |

Current behaviour
--
```php
interface TokenInterface {
	public const NEW_TOKEN_LIFETIME = 3600 * 24;
	public function generate(int $timestamp = NULL);
}

class Token implements TokenInterface {
	public function generate(int $timestamp = NULL) {
		$expired = $timestamp ?? time() + self::NEW_TOKEN_LIFETIME;
	}
}
```
Inspection warning on `$timestamp ?? time() + self::NEW_TOKEN_LIFETIME` with message `[EA] Resolved operands types are not complimentary, while they should be ([int] vs [float])`.

Expected behaviour
--
No warnings.

Environment details
---
![изображение](https://user-images.githubusercontent.com/13345595/95442522-29f16e80-0964-11eb-9508-4200cdb8b78e.png)

**Comment by ea-inspections-team** [comment]
Thanks for reporting @badmansan! Fixed!
